leenlab2/reminiscence
# Story intro design

# Description

- Changed temp intro animation for stylized video
- Added skip button

# How Has This Been Tested?

Run from the main menu scene. Try both paths:
- Press play, watch whole custscene, it should switch to main scene after
- Press play, skip cutscene, should switch to main

I am getting this console error but i have no idea where its coming from:
![image](https://github.com/leenlab2/reminiscence/assets/74102544/993b97f6-985d-4e35-8229-b9d1cdfeaaef)


# Checklists
## Integration Checklist

## Review Checklist
